corehq/apps/domain/views.py

```diff
@@ -2172,14 +2172,15 @@ def internal_settings_form(self):
             'sf_account_id',
             'services',
             'initiative',
-            'workshop_region',
+            'self_started',
             'area',
             'sub_area',
             'organization_name',
             'notes',
-            'self_started',
             'phone_model',
             'commtrack_domain',
+            'business_unit',
+            'workshop_region',
```

Comment:
ordering changed to match: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/forms.py#L762-762 for easier comparison